config/config.go

```diff
@@ -105,6 +108,8 @@ func ReadConfig(configPath string) (*Config, error) {
 		BPFLogDir:                      LoadOptionalConfigString(confReader, "l3afd", "bpf-log-dir", ""),
 		MinKernelMajorVer:              LoadOptionalConfigInt(confReader, "l3afd", "kernel-major-version", 5),
 		MinKernelMinorVer:              LoadOptionalConfigInt(confReader, "l3afd", "kernel-minor-version", 15),
+		FileLogging:                    LoadOptionalConfigBool(confReader, "l3afd", "file-logging", false),
+		FileLogLocation:                LoadOptionalConfigString(confReader, "l3afd", "file-log-location", "l3afd.log"),
```

Comment:
Can we have one option that allows the program to write to a specific file path, "log-file-path: /var/log/l3afd.log", if it is provided? Otherwise, if no file path is specified, the program should follow the default behavior of not writing to a log file.

Reply:
Fixed